### ISSUE TITLE
Versions above and including 1.17 show different colours.

### ISSUE THREAD

**Issue by IJustWantANamePlease** [ToroCraft/ToroHealth#121]
In 1.16 friendly mobs have green bars, and everything else have red ones.
However starting from 1.17 the bars change to blue and purple(or is it purple and blue). When I want to change colors manually, none of the formats seem to match the rendered colours.

**Comment by frodare** [comment]
I have seen this reported a few times now but haven't had any luck reproducing it on my computer.  By any chance are you using a Mac?  There is a known issue that should be fixed in a few days, however it might only affect Macs.

**Comment by IJustWantANamePlease** [comment]
No, I am using Windows. Both of the computers I have (both Windows) have the same problem, and only in 1.17.x. 
BTW a screenshot with only Torohealth & fabric API:
![screenshot](https://user-images.githubusercontent.com/69347951/130068890-898c5ac0-0150-4442-9bd8-34ec0c7c9023.png)

**Comment by frodare** [comment]
OK, it might still be related. I will post back here when the version goes up.  Hopefully it will fix it.  Thanks for letting me know.

**Comment by frodare** [comment]
Please try version torohealth-1.17.1-fabric-12.jar, I am curious if it will fix this issue.
